app/controllers/api/v1/schedules_controller.rb

```diff
@@ -37,20 +57,6 @@ def schedules_params
 					:thursday,
 					:friday,
 					:saturday,
-					:sundayopen,
-					:sundayclose,
-					:mondayopen,
-					:mondayclose,
-					:tuesdayopen,
-					:tuesdayclose,
-					:wednesdayopen,
-					:wednesdayclose,
-					:thursdayopen,
-					:thursdayclose,
-					:fridayopen,
```

Comment:
why are these being removed ?>

Reply:
They were added by a team member for a stretchy thingy but it was abandoned for the sake of MVP =D

Reply:
rodger that